playbooks/common/openshift-node/config.yml

```diff
@@ -79,7 +78,7 @@
   hosts: oo_nodes_to_config
   gather_facts: no
   vars:
-    sync_tmpdir: "{{ hostvars.localhost.mktemp.stdout }}"
+    sync_tmpdir: "{{ hostvars.localhost.mktemp.stdout }}/{{ groups['oo_first_master'][0] }}/{{ hostvars.localhost.mktemp.stdout }}"
```

Comment:
This (along with the change for the fetch task to remove the flat: yes parameter) is needed to support running he ansible playbook on he master host itself, otherwise, the unarchive task that is called later fails.  Without setting the flat parameter to yes for fetch, it creates the fetched file under <hostname>/<full path to file on remote> and since we are using the value of mktemp as the remote location where we are creating the tarballs on the master, we end up with this path.